pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -110,13 +176,56 @@ var assetUploadTpl = template.Must(
 	))
 
 func makeBuild(t Target, dockerfile string, opts RemoteOptions) (*cloudbuild.Build, error) {
+	var buildScript bytes.Buffer
+	uploads := []upload{
+		{From: "/workspace/image.tgz", To: opts.RemoteMetadataStore.URL(Asset{Target: t, Type: ContainerImageAsset}).String()},
+		{From: path.Join("/workspace", t.Artifact), To: opts.RemoteMetadataStore.URL(Asset{Target: t, Type: RebuildAsset}).String()},
+	}
+	if opts.UseNetworkProxy {
+		err := proxyBuildTpl.Execute(&buildScript, map[string]any{
+			"UtilPrebuildBucket": opts.UtilPrebuildBucket,
+			"DockerIPTemplate":   `{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}`,
```

Comment:
I think this is maybe a bug? It results in non-evaluated template code in the final build steps.

Reply:
nah i didn't know of a better way to embed templates in templates. this works but the printf solution you suggested offline works better. thanks!